src/App.tsx

```diff
@@ -1,14 +1,47 @@
 /* eslint-disable max-len */
-import React from 'react';
+import { FC, useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+export enum Status {
+  all = 'all',
+  active = 'active',
+  completed = 'completed',
+}
+
+export const App: FC = () => {
+  const [status, setStatus] = useState<Status>(Status.all);
+  const [query, setQuery] = useState<string>('');
+  const [todosFromServer, setTodosFromServer] = useState<Todo[]>([]);
+  const [isOpenedTodo, setIsOpenedTodo] = useState<Todo | null>(null);
```

Comment:
we start the variable name with `is/can/should/etc.` only for boolean values. I thought it is a boolean at first. What about `activeTodo`?